Gemfile

```diff
@@ -366,4 +366,6 @@ end
 
 group :production, :staging, :sandbox do
   gem 'syslog-logger'
+  # For sandbox testing
+  gem "faker"
```

Comment:
the faker gem should only be added to `staging` environment not on `production` and `staging` environments